samples/nrf_desktop/configuration/nrf52840_pca20041/battery_lut.h

```diff
@@ -0,0 +1,13 @@
+/* Converting measured battery voltage[mV] to State of Charge[%].
```

Comment:
Add copyright header and multi-include protection (just like in other xyz_def files)

Reply:
done

Reply:
done